docs/synthetics/ci-cd-integration.md

```diff
@@ -0,0 +1,300 @@
+title: CI/CD Integration
+description: Guide to how to run HTTP/Browser monitors from your CI/CD pipeline.
+
+In addition to the scheduled monitor runs, you can also trigger monitor runs using API. You can use this API to trigger monitors runs as part of your CI/CD pipeline and block the deployment if the runs fail. On failure of the run, you will be alerted on the configured alert [notification hooks](../alerts/alert-notifications.md). The run monitor API can be used to:
```

Comment:
In addition to the scheduled monitor runs, you can also trigger monitor runs using API. You can use this API to trigger monitors runs as part of your CI/CD pipeline and block the deployment if the runs fail.
=>
In addition to scheduled monitor runs, you can also trigger monitor runs by using an API. You can use this API to trigger monitor runs as part of your CI/CD pipeline and block deployments if runs fail.

Reply:
On failure of the run, you will be alerted on the configured alert [notification hooks](../alerts/alert-notifications.md). The run monitor API can be used to:
=>
When a run fails, you'll be alerted via your configured alert [notification hooks](../alerts/alert-notifications.md). The run monitor API can be used to:

Reply:
done